### PR TITLE
Simplify CI build, fix distributive build

### PR DESCRIPTION
**Pull request by siddharthist**
I messed up :( because of my lack of understanding of the CI build, I pushed a distributive package that built locally, but wouldn't in Travis. Luckily, it was never published. I've amended my error here, and clarified how the build works. Hopefully, this will prevent future errors of the same kind!

The commits are not quite atomic, but I've cleaned them up a bit. Sorry!

### CODE REVIEW

File: scripts/install.sh [hunk 1]
```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+
+if [[ $PACKAGE == "distributive" ]] && ! which godep; then
+  go get -v github.com/tools/godep
+  go build -o tools/bin/godep github.com/tools/godep
```
Comment (BrianHicks) [inline scripts/install.sh]: IIRC godep is installed in the Travis Go environment by default

**Comment by BrianHicks** [comment]
Travis uses exit code to determine if a build is marked as success or failure. This changes the pre-install exit code to 0, which means "go ahead with the rest of the process" instead of "skip the rest with success" as I think you intend.

We need the following two conditions to be met:

1. a "skipped" build is marked as success
2. a "skipped" build stops the build

Exiting 0 violates condition 2, exiting anything else violates condition 1. That's why these were bundled all inside one step. If you can find a way to satisfy both limitations without bundling, that's great. But this is not going to work as currently written.

File: scripts/install.sh (hunk 1 above)
Comment (siddharthist) [inline scripts/install.sh]: See line 313 of build: https://travis-ci.org/asteris-llc/mantl-packaging/jobs/113572361

Comment (BrianHicks) [inline scripts/install.sh]: how about that! Then memory fails me. Ignore my comment, in that case. :)

**Pull request closed without merging**